src/widgets/actionseditor.cpp

```diff
@@ -69,14 +69,11 @@ QList <QKeySequence> ActionsEditor::stringToShortcuts(QString shortcuts) {
 
 	for (int n=0; n < l.count(); n++) {
 		//qDebug("%s", l[n].toUtf8().data());
-#if QT_VERSION >= 0x040300
+
```

Comment:
Also remove the comment regarding Qt 4.3 and 4.4 versions below.

Reply:
Is that enough?
``` diff
-               // Qt 4.3 and 4.4 (at least on linux) seems to have a problem when using Traditional Chinese
+               // Qt (at least on linux) seems to have a problem when using Traditional Chinese
```
And is the whole workaround still needed with Qt5?

Reply:
@suamor Should this be cleaned up on Qt5? What "arrow keys" did `QKeySequence` delete?

Reply:
@ff2000: go ahead and apply this comment fix. We will open the issue later, if the error is still there.